S/sais/build_tarballs.jl

```diff
@@ -36,13 +31,16 @@ platforms = supported_platforms()
 
 
 # The products that we will ensure are always built
-products = [
-    LibraryProduct("libsais", :libsais)
-]
+libsais = LibraryProduct("libsais", :libsais)
```

Comment:
Can we just have the same product with two different possible names for the library? That would make everything much easier here and also on the side of the julia package

Reply:
Alas libsais64 is not a superset of libsais, it contains only the extra functions for 64-bit platforms. It's not my library so I can't change it. I could make an empty libsais64 on 32-bit platforms?

Reply:
> Alas libsais64 is not a superset of libsais, it contains only the extra functions for 64-bit platforms.

:sob:

Reply:
If it's a pain it can be left as it is --- I just thought it was nicer to keep to the original design of the library instead of hacking around with how it is build.

Reply:
It's possible to have different products for different platforms but it requires a bit more work, I can have a look later today.

However, different products on different platforms means that on the julia side you need to check on which platform you're to guess which libraries are available. This basically adds more complexity here and on the julia side.  Is it really worth the extra pain everywhere? L

Reply:
> on the julia side you need to check on which platform you're to guess which libraries are available

You need to do that anyway in order to use the extra functionality that libsais64 provides, i.e. libsais64 contains extra functions which accept 64-bit integers.

Reply:
Yet another case where some kind of "forwarding" library would be useful; e.g. to add the symbols `foo64 -> foo` to libsais64.